source/common/http/filter_manager.h

```diff
@@ -1012,22 +1012,24 @@ class FilterManager : public ScopeTrackedObject,
     struct FilterCallState {
       static constexpr uint32_t DecodeHeaders   = 0x01;
       static constexpr uint32_t DecodeData      = 0x02;
-      static constexpr uint32_t DecodeTrailers  = 0x04;
-      static constexpr uint32_t EncodeHeaders   = 0x08;
-      static constexpr uint32_t EncodeData      = 0x10;
-      static constexpr uint32_t EncodeTrailers  = 0x20;
+      static constexpr uint32_t DecodeMetadata  = 0x04;
```

Comment:
Why shift these values? I think it was nice they were in order but the requirement really is they're unique.

Reply:
I just wanted them in order. No other reason. Happy to revert if you prefer.